tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -367,3 +375,15 @@ def test_mo_import_from_memory_tf_fe(self, create_model, ie_device, precision, i
         if mo_params is not None:
             test_params.update(mo_params)
         self._test_by_ref_graph(temp_dir, test_params, graph_ref, compare_tensor_names=False)
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit_tf_fe
+    @pytest.mark.precommit
+    def test_unnamed_saved_model_dir(self, ie_device, precision, ir_version, temp_dir):
+        saved_model_dir, graph_ref = create_tf_saved_model_dir(temp_dir)
+
+        test_params = {'input_model': saved_model_dir, 'use_new_frontend': True}
+        self._test_by_ref_graph(temp_dir, test_params, graph_ref, compare_tensor_names=False)
+
```

Comment:
Will we support cmd-line without specifying `--input_model`: `mo <saved_model_dir>`?

Reply:
No, here I add support only for Python API.